src/startupscriptgenerator/src/common/versionInfo.go

```diff
@@ -5,14 +5,19 @@
 
 package common
 
+import (
+	"fmt"
+)
+
 // NOTE: The 'BuildNumber' and 'Commit' variables here are modified during build time. Look at
 // the 'go build' command in file 'startupscriptgenerator/build.sh'
 var (
-	BuildNumber = "unspecified"
-	Version     = "0.2" + "." + BuildNumber
-	Commit      = "unspecified"
+	BuildNumber    = "unspecified"
+	Version        = "0.2" + "." + BuildNumber
+	Commit         = "unspecified"
+	ReleaseTagName = "unspecified"
 )
 
 func PrintVersionInfo() {
-	println("Oryx Version : " + Version + ", Commit: " + Commit + "\n")
+	fmt.Printf("Oryx Version: %s, Commit: %s, ReleaseTagName: %s\n", Version, Commit, ReleaseTagName)
```

Comment:
tidbit: it seems `println` writes to `stderr` where as `fmt.Printf` here writes to `stdout`